locales/de.yml

```diff
@@ -0,0 +1,248 @@
+de:
+   rodauth:
```

Comment:
There appears to be three spaces of indentation here instead of two.

Reply:
yeiks, of course.